src/app/earn/use-formatted-data.tsx

```diff
@@ -78,14 +82,28 @@ export function useFormattedEarnData(): FormattedEarnData {
     gasFeesEth =
       gasCosts < BigInt(1000000000000000)
         ? '(<0.001 ETH)'
-        : `(${formatWei(gasCosts, 18)} ETH)`
+        : `(${Number(formatWei(gasCosts, 18)).toFixed(2)} ETH)`
   }
 
   const shouldShowSummaryDetails = useMemo(
     () => quote !== null && inputValue !== '',
     [inputValue, quote],
   )
 
+  let orderFee = ''
```

Comment:
just note/nit: you can put the expression below into a useMemo, and return these to avoid the let declaration and value assign